package.json

```diff
@@ -60,46 +60,41 @@
     "onCommand:PowerShell.SpecifyScriptArgs"
   ],
   "dependencies": {
-    "@vscode/extension-telemetry": "^0.9.8",
+    "@vscode/extension-telemetry": "^0.9.9",
     "node-fetch": "^2.7.0",
-    "semver": "^7.7.1",
+    "semver": "^7.7.2",
     "untildify": "^4.0.0",
     "uuid": "^9.0.1",
     "vscode-languageclient": "^9.0.1",
     "vscode-languageserver-protocol": "^3.17.5"
   },
   "devDependencies": {
-    "@vscode/vsce": "^3.3.0",
-    "esbuild": "^0.25.1"
+    "@vscode/vsce": "^3.3.2",
+    "esbuild": "^0.25.4"
   },
   "optionalDependencies": {
```

Comment:
We probably should document *somewhere* why optionalDependencies vs devdependencies

Reply:
I'm really hoping it changes when I get back to trying #5063